### PR TITLE
Added refinement.center_refinement(...)

### PR DESCRIPTION
**Pull request by VikingScientist**

### CODE REVIEW

**Comment by VikingScientist** [comment]
How do I provoke a travis test on this PR?

**Comment by TheBB** [comment]
You can trigger builds on branches but I'm not sure you can do it on PRs. If you push any change at all it should build again though. For example you can rebase on the new master.

File: test/utils/refinement_test.py [hunk 1]
```diff
@@ -78,6 +78,18 @@ def test_edge_volume(self):
         for low,high in zip(knots, knots[::-1]):
             self.assertAlmostEqual(low, 1-high)
 
+    def test_center_volume(self):
+        vol = Volume()
+        vol.raise_order(1,2,3)
+        edge_refine(vol, 1.3, 6, direction=2)
```
Comment (TheBB) [inline test/utils/refinement_test.py]: Was the idea to test `center_refine` here?

Comment (VikingScientist) [reply test/utils/refinement_test.py]: Yes it was 😊 Will fix and update PR

**Comment by VikingScientist** [comment]
Fixed and ready for merging @TheBB

**Review by TheBB — APPROVED** [review]

**Pull request merged**